<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadUrlFileDemoClass</class>
 <widget class="QMainWindow" name="DownloadUrlFileDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownloadUrlFileDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>URL：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUrl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>http://qunying.jb51.net:81/201011/books/gof23_pdf.rar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownload">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>下载并打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>下载进度：</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbDownload">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DownloadUrlFileDemo.qrc"/>
 </resources>
 <connections/>
</ui>
